src/components/ShowUser/RenderUser/RenderUser.jsx

```diff
@@ -35,66 +37,89 @@ function RenderUser(props) {
 					</Typography>
 				) : t('unavailable')}
 				{props.user.email != null ? (
-					<>
-						<Typography variant="subtitle1" color="textPrimary">
-							{t('email')}
-						</Typography>
-						<Typography variant="caption" color="textSecondary">
-							{props.user.email}
-						</Typography>
-					</>
-				) : null }
+					<Typography variant="body2" component="p">
+						{t('email')}: {props.user.email}
+					</Typography>
+				) : null}
 				{props.user.dateOfBirth != null ? (
-					<>
-						<Typography variant="subtitle1" color="textPrimary">
-							{t('dateOfBirth')}
-						</Typography>
-						<Typography variant="caption" color="textSecondary">
-							{props.user.dateOfBirth}
-						</Typography>
-					</>
-				) : null }
+					<Typography variant="body2" component="p">
+						{t('dateOfBirth')}: {props.user.dateOfBirth}
```

Comment:
date of birth skrives som 2019-11-27T00:00:00.000Z, måske skal det formateres lidt